### ISSUE TITLE
No module named zoo

### ISSUE THREAD

**Issue by waalge** [fireshape/fireshape#65]
Similar to https://github.com/fireshape/fireshape/issues/53, maybe 

Trying to the run the cantilever example found in https://zenodo.org/record/4036176
```
$python main.py 
Traceback (most recent call last):
  File "main.py", line 2, in <module>
    from fireshape import *
  File "/home/waalge/clones/firedrake/lib/python3.8/site-packages/fireshape/__init__.py", line 8, in <module>
    import zoo
ModuleNotFoundError: No module named 'zoo'
```
Context: 
I'd run 
```
pip install git+https://github.com/fireshape/fireshape.git
```
TBH i don't know which version that installs. 

## Fixed with:
Uninstalling and using the cloned repo for the install fixed the error
```
pip install -e . 
```

**Comment by rarygit** [comment]
Thanks @waalge!

That solution helped to start running "python3 main_pipe.py >> output.txt" 

Could also add the advice to run command 'make' in the directory ~/examples/pipe  to convert .geo file to .msh

**Comment by APaganini** [comment]
Fixed in #76
